teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -55,6 +56,24 @@ public class ValidatorOptions {
       arity = "1")
   private Path graffitiFile;
 
+  @Option(
+      names = {"--validators-graffiti-client-append-format"},
+      paramLabel = "<STRING>",
+      description =
+          """
+                          Appends CL and EL clients information separated with a space to user's graffiti on the Beacon Node.
+                          AUTO modes tries to provide complete version information if there is enough space for it.
+                          To disable use NONE. Available options:
+                            AUTO_END: (default) Clients info is appended after user's graffiti if any.
+                            AUTO_START: Clients info is added before user's graffiti.
+                            NAME_END: Only clients names are appended after user's graffiti. Good when you don't want to disclose clients versions.
+                            NAME_START: Clients names are added before user's graffiti.
+                            NONE: Clients information is not added to the graffiti.
```

Comment:
i'd basically only offer
AUTO - which applies client name and version tag to graffiti, and is default
CLIENT_NAME - allows us to apply client name only to the graffiti. This would be greatly appreciated so that client information can be more accurate kind of description
DISABLED - would be to turn this feature off

worth checking other implementation teams to see their plans if we haven't...
also i think we can actually use enum in cli args which may make this whole thing a little cleaner rather than making it string...